src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1756,4 +1756,7 @@ This version used in: {2}</value>
   <data name="Implement_explicitly" xml:space="preserve">
     <value>Implement explicitly</value>
   </data>
+  <data name="Struct_contains_assigment_to_comma_this_comma_outside_the_scope_constructor_Make_readonly_fields_writable" xml:space="preserve">
```

Comment:
Remove the _comma_ parts (we don't include punctuation, and those aren't commas either)

Reply:
ok